lib/stages.js

```diff
@@ -0,0 +1,21 @@
+const STAGES = [
+  {
+    name: 'review',
+    inferRegex:  /-pr-(\d+)$/
+  },
+  {
+    name: 'development',
+    inferRegex: /-(dev|development)$/
```

Comment:
Might I suggest that you map the testing and QA patterns into here too? That way they don't show up as production apps.